docs/stylesheets/extra.css

```diff
@@ -50,6 +56,33 @@
   --md-admonition-bg-color: var(--md-default-bg-color);
 }
 
+/* Header color */
+.md-header {
+    background-color: var(--md-default-bg-color);
+    color: var(--md-typeset-color);
+}
+.md-tabs {
+    background-color: var(--md-default-bg-color);
+    color: var(--md-typeset-color);
+    border-bottom: 1px solid var(--md-border-color);
+}
+.md-search__form:hover {
+    background-color: #0000001f;
+}
+
+/* Site title and logo styling */
+.md-header__button:hover {
+    opacity: 1.0;
+}
+.md-header__topic:first-child {
+    font-weight:400;
+}
+[dir="ltr"] .md-header__title {
+    margin-left: 0.0;
+    font-size:.95rem;
```

Comment:
Usually tabs = 2 spaces in CSS, and there's a space between the comma following argument name `:` and the value. The formatting is uneven in this file (you may want to turn on auto-format on save in your IDE).